src/app/lib/app-switcher/app-switcher.component.ts

```diff
@@ -16,6 +16,7 @@ export class AppSwitcherComponent implements OnInit, OnDestroy {
     public subscription: Subscription;
 
     private ngUnsubscribe: any = new Subject();
+    private brandBg = 'brand';
```

Comment:
you need to make this public to pass the travis ci check. private properties cannot be accessed by the markup 

`app-switcher.component.html(5,6): : Property 'brandBg' is private and only accessible within class 'AppSwitcherComponent'`